wiki/Behavior.md

```diff
@@ -26,3 +26,47 @@ The name is generated as follows:
 See the [Migration System] page.
 
 [Migration System]: https://github.com/recyclarr/recyclarr/wiki/Migration-System
+
+## Radarr
+
+### Custom format synchronization is broken up into three categories
+
+- Creation: Custom formats that are in the guide but do not exist in Radarr are **created**.
+- Updates: Custom formats that already exist in both the guide and in Radarr are **updated**.
+- Deletions: If deletions are allowed by having the `delete_old_custom_formats` configuration
+  setting set to `true`, then custom formats in Radarr are deleted if they are removed from the
+  guide **or** removed from your configuration file.
+
+> **Important**
+>
+> Recyclarr will *never* touch custom formats that you create by hand, unless they share a name with
+> a custom format in the guide. In general, Recyclarr must have been the one to create a custom
+> format in order to do anything to it (update or delete).
+
+#### Cache
```

Comment:
I know you were keeping the heading nesting as close as possible to the original. But I think because we're deeper into this, it might be better to instead use `###` here and drop down 1 level for the nested headings (i.e. `#### Summary`)

Reply:
I Agree, I'll make the change and repush.